<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>assignment</class>
 <widget class="QDialog" name="assignment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(211, 215, 207)</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>340</y>
     <width>281</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>240</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>You can select assignment files in here</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_save">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>390</y>
     <width>89</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_goto">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>500</y>
     <width>89</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Go to</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_select">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>240</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>300</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Assignment Description</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>80</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Assignment No.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Subject</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Level</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Semester</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_assignmentno">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>110</y>
     <width>86</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_sem">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>180</y>
     <width>86</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_subject">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>100</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Computer Programming</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Introduction to Computers and Operating Systems</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Java Programming</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Object Oriented Programming</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Database Management Systems</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Data Structures and Algorhythms</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Computer Architecture and Compiler Design</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rapid Application Development</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Data communication and Computer Networks</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Software Engineering</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Web Designing and e-commerce</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mobile and Cloud Computing</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Image Processing</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Artificial Intelligence</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Parallel Computing</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Data Mining</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_level">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>140</y>
     <width>86</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;vnvbnvbn&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#a40000;&quot;&gt;vnb vbnvnn&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#204a87;&quot;&gt;Add Assignment&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>200</y>
     <width>110</width>
     <height>26</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2019</year>
     <month>2</month>
     <day>21</day>
    </datetime>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>170</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Assignment Date </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
